src/sql/src/kafka_util.rs

```diff
@@ -13,9 +13,11 @@ use std::collections::BTreeMap;
 use std::convert;
 use std::fs::File;
 use std::io::Read;
+use std::option::Option;
 use std::sync::Mutex;
 
 use anyhow::bail;
+use chrono;
```

Comment:
These imports look like cruft from whatever your IDE is? No need to import either! The first is in the standard prelude, and the second is a crate root so implicitly in scope.